operations/messaging-and-math/content-marketing/video-content.md

```diff
@@ -0,0 +1,86 @@
+---
+description: '50% DRAFT (revised by Ashley Dotterweich on Oct 20, 2020)'
+---
+
+# Video Content Best Practices
+
+This document provides guidelines for creating video content for Mattermost. It covers:
```

Comment:
The bulletpoints don't match up with the subheadings below. Options:
1. Rename them to match the headings in the content.
2. Make them sentences introduced by: "The guidelines should give you an idea of how to: "Create great content", Aim for the correct audience", Record high-quality videos" etc. Then it's not prescriptive but also flows with the ideas below.

Reply:
I like #2 -- if I remember correctly, my goal was to provide a high-level overview of what's covered, rather than a page-level table of contents.